less/index.less

```diff
@@ -54,6 +54,138 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	background: #659DBD;
 }
 
+@mobile-width: 500px;
+
+.flex-row-nowrap {
+	display: flex;
+	flex-flow: row nowrap;
+}
+
+.flex-column-nowrap {
+	display: flex;
+	flex-flow: column nowrap;
+}
+
+.container {
+	width: 900px;
+	margin: 0 auto;
+	font-family: 'Sarabun', sans-serif;
+
+	@media (max-width: @mobile-width) {
+		width: @mobile-width;
+	}
+
+	header {
+		.flex-column-nowrap();
+	
+		nav {
+			.flex-row-nowrap();
+			justify-content: space-evenly;
+			align-content: center;
+			margin: 20px 0px;
+
+			@media (max-width: @mobile-width) {
+				flex-flow: column nowrap;
+			}
+		}
+
+		nav a {
+			color: gray;
```

Comment:
I would the color of the font because it makes it hard to see with the color of your background